From c5d5725e6040e323b79a0672b7c97c615d2e857f Mon Sep 17 00:00:00 2001
From: Bruno Martins <bgcngm@gmail.com>
Date: Tue, 5 Dec 2017 00:26:35 +0000
Subject: [PATCH 1/2] AppOpsManager: Update with the new ops

 * Includes new ops introduced in Oreo as well as ours

Change-Id: I713bbdee1da9d142cf9671128fa05391fa38c48a
---
 libs/binder/include/binder/AppOpsManager.h | 18 ++++++++++++++++++
 1 file changed, 18 insertions(+)

diff --git a/libs/binder/include/binder/AppOpsManager.h b/libs/binder/include/binder/AppOpsManager.h
index 4212776..1beabd3 100644
--- a/libs/binder/include/binder/AppOpsManager.h
+++ b/libs/binder/include/binder/AppOpsManager.h
@@ -92,7 +92,25 @@ public:
         OP_PROCESS_OUTGOING_CALLS = 54,
         OP_USE_FINGERPRINT = 55,
         OP_BODY_SENSORS = 56,
+        OP_READ_CELL_BROADCASTS = 57,
+        OP_MOCK_LOCATION = 58,
+        OP_READ_EXTERNAL_STORAGE = 59,
+        OP_WRITE_EXTERNAL_STORAGE = 60,
+        OP_TURN_SCREEN_ON = 61,
+        OP_GET_ACCOUNTS = 62,
+        OP_RUN_IN_BACKGROUND = 63,
         OP_AUDIO_ACCESSIBILITY_VOLUME = 64,
+        OP_READ_PHONE_NUMBERS = 65,
+        OP_REQUEST_INSTALL_PACKAGES = 66,
+        OP_PICTURE_IN_PICTURE = 67,
+        OP_INSTANT_APP_START_FOREGROUND = 68,
+        OP_ANSWER_PHONE_CALLS = 69,
+        OP_CHANGE_WIFI_STATE = 70,
+        OP_BLUETOOTH_CHANGE = 71,
+        OP_BOOT_COMPLETED = 72,
+        OP_NFC_CHANGE = 73,
+        OP_DATA_CONNECT_CHANGE = 74,
+        OP_SU = 75
     };
 
     AppOpsManager();
-- 
2.7.4

